src/table/table.scss

```diff
@@ -129,11 +129,18 @@
   display: flex;
   flex-direction: column;
   flex-grow: 1;
-
   @include themed {
     background-color: t(iui-color-background-1);
   }
 
+  &.iui-scroll-snapping {
```

Comment:
How about I put this in `/style/global.scss`:
```
.iui-scroll-snapping {
  scroll-snap-type: y mandatory;
}
```
And then leave `scroll-snap-align: start;` within the general `.iui-row` block?  This way the `.iui-scroll-snapping` class could be used in other places (maybe [menu](https://itwin.github.io/iTwinUI/backstop/minified/menu.html) & [codeblock](https://itwin.github.io/iTwinUI/backstop/minified/code.html)).  Those components would need to add the `scroll-snap-align: start;` part though.

Reply:
maybe in `mixins.scss` not `global.scss`

Reply:
or both actually. create a mixin and use it in a class in `global.scss`

similar to iui-anchor
![image](https://user-images.githubusercontent.com/9084735/141184683-3e12306f-cd3e-4022-b259-6b00e915bdb0.png)

Reply:
The problem is that global css is included in every single component/file, so we don't want to add unnecessary stuff there.

Maybe @veekeys has a different suggestion

Reply:
Global should not consist of just plain mixins, IMO. There should be left minimal needed rules for itwinui to look better (so global css reset, scroll, etc..).
If we want just to have some helper mixins, this is also good, but lets put them in separate file, which would be included only in demand. Also, if it gets big, we might want to put it in chunks even more.
I would also suggest to start thinking about expanding our documentation, havnig more readme or w/e, because we already have a bunch of really good stuff, but it is hard to find or even remember (personally).

Reply:
Just checked the implementation now..
One question, without `scroll-snap-align: start` snapping wont work at all?? Like it is mandatory?
Also, I am not sure about mixin now :D That would require including it explicitly in every component, where you actually need. Instead, I would think it should be enough to include css with that class in your app and you can use that class wherever you need (wouldnt that be more comfortable??). I am talking here from users perspective. But it still should not be in global (forced include).

Reply:
> One question, without scroll-snap-align: start snapping wont work at all?

Yes, because the browser doesn't know where to snap.

> That would require including it explicitly in every component, where you actually need.

Not a bad thing imo. It gives us the control to only provide snapping where we approve.

> I would think it should be enough to include css with that class in your app and you can use that class wherever you need

I was thinking the user would just toggle a react prop in the components we add this to. We can't expect them to find this class name, and it's not something difficult to write on your own.

Reply:
In this case, mixin of a single rule looks kinda useless...

> I was thinking the user would just toggle a react prop in the components we add this to. We can't expect them to find this class name, and it's not something difficult to write on your own.

You are talking here about react users, whereas I had in mind css users and the documentation part too. I do agree about props and our components, but I do feel scroll snapping set of rules might be something we could advertise and provide from css package as well.